src/components/movie-component/movie-component-2.js

```diff
@@ -0,0 +1,23 @@
+class MovieComponent2 extends HTMLElement {
+  connectedCallback() {
+    this.innerHTML = `
+        <div>
+        <div class="movie">
+        <div class="opacity">
+        <div class="info">
+        <div class="rating-stars">
```

Comment:
dlaczego tak dużo tu divów? 5 obok siebie to już naprawdę sporo. Możesz to zredukować?